api/src/controllers/users.ts

```diff
@@ -44,6 +46,7 @@ router.get('/auth/google',
  * Callback for Google authentication
  */
 router.get('/auth/google/callback', function (req, res) {
+  console.log(req.session);
```

Comment:
If we're not using it could you comment this line out to minimize log sizes?

Reply:
Done